examples/examples.json

```diff
@@ -0,0 +1,234 @@
+              {
+                "propertyName": "env",
+                "operator": "EQ",
+                "valueToMatch": {
+                  "string": "staging"
+                }
+              }
+            ],
+            "value_on_match": {
+              "log_level": "WARN"
+            }
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "key": "feature-flag",
+    "allowed_values": [
+      {
+        "string": "treatment"
+      },
+      {
+        "string": "control"
+      }
+    ],
+    "config_rows": [
+      {
+        "properties": {
+          "active": true
+        },
+        "rules_and_values": [
+          {
+            "criterias": [
+              {
+                "propertyName": "env",
+                "operator": "EQ",
+                "valueToMatch": {
+                  "string": "staging"
+                }
+              },
+              {
+                "propertyName": "active",
+                "operator": "EQ",
+                "valueToMatch": {
+                  "boolean": true
+                }
+              },
+              {
+                "propertyName": "email",
+                "operator": "ENDS_WITH",
+                "value": "example.com"
+              }
+            ],
+            "value_on_match": {
+              "weighted": [
+                {
+                  "variant": {
+                    "string": "treatment"
+                  },
+                  "weight": 0.5
+                },
+                {
+                  "variant": {
+                    "string": "control"
+                  },
+                  "weight": 0.5
+                }
+              ]
+            }
+          },
+          {
+            "criterias": [
+              {
+                "propertyName": "env",
+                "operator": "EQ",
+                "valueToMatch": {
+                  "string": "staging"
+                }
+              },
+              {
+                "propertyName": "active",
+                "operator": "EQ",
+                "valueToMatch": {
+                  "boolean": false
+                }
+              }
+            ],
+            "value_on_match": {
+              "string": "control"
+            }
+          },
+          {
+            "criterias": [
+              {
+                "propertyName": "env",
+                "operator": "EQ",
+                "valueToMatch": {
+                  "string": "staging"
+                }
+              },
+              {
+                "propertyName": "active",
+                "operator": "EQ",
+                "valueToMatch": {
+                  "boolean": true
+                }
+              },
+              {
+                "operator": "IN_SEGMENT",
+                "valueToMatch": {
+                  "config_value": "alpha-customer-segment"
+                }
+              }
+            ],
+            "value_on_match": {
+              "string": "alpha-treatment"
```

Comment:
@jdwyah If I'm following this right, I think `alpha-treatment` would need to be added to the list of `allowed_values`, since this is all the same feature flag

Reply:
oo. good eyes. yep.